test/performance/odp_sched_latency.c

```diff
@@ -255,7 +259,7 @@ static int enqueue_events(int prio, int num_queues, int num_events,
  *
  * @param globals  Test shared data
  */
-static void print_results(test_globals_t *globals)
+static int output_results(test_globals_t *globals)
```

Comment:
All applications could use the same function name for outputting the results. I'd perhaps prefer this one (`output_results()`).